.github/workflows/main.yml

```diff
@@ -30,7 +42,7 @@ jobs:
 
   cdk-deploy-prod:
     if: ${{github.ref == 'refs/heads/prod'}}
-    needs: tests
+    needs: [tests, synth-test-dev, synth-test-prod]
```

Comment:
same here, does this need `synth-test-dev`?

Reply:
Good point.  I have updated the PR accordingly.